src/model/related-article.ts

```diff
@@ -0,0 +1,40 @@
+import { BackmatterEntity } from './backmatter-entity';
+import {JSONObject} from "./types";
+
+export class RelatedArticle extends BackmatterEntity {
+  public articleType: string = '';
+  public href: string = '';
+
+  constructor(data?: JSONObject | Element) {
+    super();
+    this.createEntity(data);
+  }
+
+  public clone(): RelatedArticle {
```

Comment:
Could we get some tests to ensure the returned  object has the same internal properties as the original but is not the same instance of the object.